pkg/resources/gateway/alerting.go

```diff
@@ -137,29 +170,34 @@ func (r *Reconciler) alerting() []resources.Resource {
 										},
 									},
 								},
+								{
+									Name:  "USER",
```

Comment:
Where is this used?

Reply:
a library that alert manager uses requires this to be explicitly set, so when we build its `main()` from source the command will exit and AlertManager will never start in the pod